docs/17-porting-tips.md

```diff
@@ -94,9 +94,22 @@ work. Remove this section.
     - end
     --- }}}
 
+The `quit` menu command must be wrapped in a function, otherwise an error
+occurs due to mismatched argument types from the v4.0 `awful.menu` library.
+
+     -- {{{ Menu
+     -- Create a laucher widget and a main menu
+     myawesomemenu = {
+       { "manual", terminal .. " -e man awesome" },
+       { "edit config", editor_cmd .. " " .. awesome.conffile },
+       { "restart", awesome.restart },
+    -  { "quit", awesome.quit }
+    +  { "quit", function() awesome.quit() end}
```

Comment:
Yes, understood - but then it makes sense to keep the context as with 3.5.x, i.e. do not fix the typo in the context.